archive/qt_installer_framework.ksy

```diff
@@ -0,0 +1,225 @@
+meta:
+  id: qt_installer_framework
+  title: Qt Installer Framework archive
+  application: Qt Installer Framework
+  file-extension: exe
+  endian: le
+  encoding: utf-8
+  license: GPL-3.0  # With exceptions, see https://github.com/qtproject/installer-framework/blob/master/LICENSE.GPL3-EXCEPT
```

Comment:
Looks like this exception is explicitly listed under the SPDX license exceptions, see https://spdx.org/licenses/Qt-GPL-exception-1.0.html.

Therefore I think there's some way how to express the _"`GPL-3.0` with `Qt-GPL-exception-1.0`_ using a SPDX expression, but I don't know how it would look like. Possibly using the `WITH` operator, as the https://spdx.org/licenses/ page suggests?

> - [**License Exceptions**](https://spdx.org/licenses/exceptions-index.html): The list of commonly found exceptions to free and open source licenses, which can be used with the License Expression operator, "WITH" to create a license with an exception.

Reply:
Thanks, fixed.